tests/testthat/test-workflows.R

```diff
@@ -142,13 +142,13 @@ test_that("workflow summary returns ref, instance and parameters", {
   reports <- list(
     list(name = "depend4",
          instance = "production",
-         parameters = list(nmin = 0.5, another_param = "test"))
+         params = list(nmin = 0.5, another_param = "test"))
```

Comment:
Rob and I chatted about how we can make these mocks a bit more robust to schema changes and he's going to take a look